### DISCUSSION TITLE
Mitoinstaller on PyPi

### DISCUSSION THREAD

**Discussion by lalyndsighart** [mito-ds/mito#441] [General]
Hello there!
I've seen Mito demo'd on youtube etc and would love to get some colleagues to try it.  However there is no link to this github repository and no github stats shown on the page https://pypi.org/project/mitoinstaller/.  Will this be updated?
Thanks,
Lalyndsighart

**Comment by aarondr77** [comment]
Hey @lalyndsighart thanks for posting. Yes, we can. I opened a PR [here](https://github.com/mito-ds/monorepo/pull/442) and it'll be released this week. 

I'd love to help your colleagues get started with Mito. Happy to share how we've supported other companies. If you send me an email to aaron@sagacollab.com, we can hop on a call and chat more.
